packages/SwingSet/test/test-controller.js

```diff
@@ -118,7 +119,9 @@ test('bootstrap', async t => {
   const c = await buildVatController(config);
   for (let i = 0; i < 5; i += 1) {
     // eslint-disable-next-line no-await-in-loop
-    await c.step(); // vat starts
+    await c.step(); // vat start acceptance
+    // eslint-disable-next-line no-await-in-loop
+    await c.step(); // vat start deliver
```

Comment:
@gibson042 has a PR #4678 that's about to land that gets rid of some of this fussy `c.step()` stuff, which should subsume the changes you've had to make here.